### PR TITLE
Reference Neo.dll in Neo.Compiler.MSIL

### PR DESCRIPTION
**Pull request by erikzhang**

### CODE REVIEW

File: Neo.Compiler.MSIL.UnitTests/Utils/NeonTestTool.cs [hunk 1]
```diff
@@ -61,11 +61,7 @@ public NeonTestTool(string filename)
             }
 
             converterIL = new ModuleConverter(log);
-            ConvOption option = new ConvOption()
-            {
-                useNep8 = true,
-                useSysCallInteropHash = true
-            };
+            ConvOption option = new ConvOption();
```
Comment (vncoelho) [inline Neo.Compiler.MSIL.UnitTests/Utils/NeonTestTool.cs]: compatibility deprecated, right?
@igormcoelho

**Comment by vncoelho** [comment]
@erikzhang, before merging maybe we could bring neoj and neoa to this repository as well, in order to keep branches more similar as possible before incompatible changes.

File: Neo.Compiler.MSIL.UnitTests/Utils/NeonTestTool.cs (hunk 1 above)
Comment (erikzhang) [reply Neo.Compiler.MSIL.UnitTests/Utils/NeonTestTool.cs]: NEP-8 is obsolete in NEO 3.0. we use `SYSCALL System.Contract.Call` now.

**Comment by erikzhang** [comment]
`neoa` is useless now. And `neoj` could be merged into `neo-devpack-java`. But no one is currently maintaining it.

**Comment by igormcoelho** [comment]
Very nice unification Erik!! This will help a lot the maintainance of both projects.

**Comment by igormcoelho** [comment]
Vitor, we can test this new structure on neocompiler.io

**Comment by shargon** [comment]
Will it be cleaned in another pull request `TransactionInput`` TransactionOutput`, etc.?

**Comment by erikzhang** [comment]
> Will it be cleaned in another pull request TransactionInput TransactionOutput, etc.?

Yes. This PR is for compiler.

**Pull request merged**